src/app/issues/PageIssues.tsx

```diff
@@ -488,48 +496,191 @@ export const PageIssues = () => {
+                                </LinkOverlay>
+                              </Text>
+                              <Text
+                                fontSize="sm"
+                                color="gray.500"
+                                _dark={{ color: 'gray.400' }}
+                                noOfLines={2}
                               >
-                                {scope.name}
-                              </Tag>
-                            ))}
-                          </Wrap>
-                        </DataListCell>
+                                {issue.description}
+                              </Text>
+                              <Wrap>
+                                {issue.scopes?.slice(0, 1).map(({ scope }) => (
+                                  <Tag
+                                    key={scope.id}
+                                    color={
+                                      generateSwatch(
+                                        scope.color ?? brandColor
+                                      )[700]
+                                    }
+                                    bg={
+                                      generateSwatch(
+                                        scope.color ?? brandColor
+                                      )[100]
+                                    }
+                                    _dark={{
+                                      color: generateSwatch(
+                                        scope.color ?? brandColor
+                                      )[50],
+                                      bg: generateSwatch(
+                                        scope.color ?? brandColor
+                                      )[700],
+                                    }}
+                                  >
+                                    {scope.name}
+                                  </Tag>
+                                ))}
+
+                                {issue.scopes.length > 1 ? (
+                                  <Tooltip
+                                    hasArrow
+                                    p={2}
+                                    borderRadius={8}
+                                    bg="gray.800"
+                                    placement="bottom"
+                                    label={
+                                      <Wrap>
+                                        {issue.scopes
+                                          ?.slice(1, issue.scopes.length)
+                                          .map(({ scope }) => (
+                                            <Tag
+                                              key={scope.id}
+                                              color={
+                                                generateSwatch(
+                                                  scope.color ?? brandColor
+                                                )[700]
```

Comment:
Two things :

1. Here you calculate every time a new swatch from the same color, it's better to declare one time the `generateSwatch(scope.color ?? brandColor)` and then use the result each time we need a color tint
2. In this case, we should not defined manually color, bg, dark stuff... You can use colorScheme props instead